contracts/earn/maxi/Trisolaris/MaxiVault.sol

```diff
@@ -0,0 +1,253 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.6.12;
+
```

Comment:
`pragma experimental ABIEncoderV2;` will get compile error when using specific solidity feature, not need to using here.